docs/dev/contributing.md

```diff
@@ -290,6 +290,7 @@ labels that can be use:
   Python version shipped in the agent).
 - `need-change/operator`, `need-change/helm`: indicate that the configuration needs to be modified in the operator / helm chart as well.
 - `k8s/<min-version>`: indicate the lowest Kubernetes version compatible with the PR's feature.
+- `backport/<branch-name>`: Add this label to have your changes automatically backported to <branch-name>
```

Comment:
```suggestion
- `backport/<branch-name>`: Add this label to have your changes automatically backported to `<branch-name>`
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It's not displayed well without the backticks:
![image](https://github.com/DataDog/datadog-agent/assets/47357713/85da2c43-5bfa-4441-a414-2dd5343942a2)